jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/roles/CaseRoleItemView.html

```diff
@@ -24,6 +24,10 @@
                         <span class="fa fa-users"></span>
                         <div data-field="line-groups"></div>
                     </div>
+                    <div data-field="div-unassigned" class="hidden kie-full-width kie-single-role-assignment-height kie-unassigned-color ">
+                        <span class="fa fa-user"></span>
+                        <span data-i18n-key="Unassigned"></span>
```

Comment:
please add a small space between the icon and the string, adding _padding-left: 5px;_ looks good.